### DISCUSSION TITLE
Add using tags as a display rule

### DISCUSSION THREAD

**Discussion by dmantisk** [chrisgurney/obsidian-note-toolbar#83] [Ideas]
My request is to give an option to have the toolbar display only on notes with certain tags.
Ideally being able to display a specific toolbar for one or more tags.

Tags feels like the obvious way to do this, is there a reason you didn't use it?

**Comment by chrisgurney** [comment]
@dmantisk Thanks for the idea!

In Note Toolbar's settings, you can actually change the property from `notetoolbar` to `tags` and it will work with tags (assuming you name your toolbars according to those tags).

But I'm also looking into adding a mapping type specifically for tags, so it can be combined with the `notetoolbar` property if people still want to use that to override toolbars on specific notes.

**Reply by dmantisk** [reply]
I didn't realize that I could have used it like that.

I specifically want to integrate this with another plugin (spaced repetition), and while I can add another tag I'd prefer to use the existing tags. (Since the I'm using the specific toolbar _only_ for that specific plugin's commands)

Currently it requires me to create duplicate toolbars if I want to use the same toolbar setup for multiple tags.

I'm not a programmer, so I don't know the feasibility/difficulty involved, but do you think it's possible map a single toolbar for multiple tags?

**Reply by chrisgurney** [reply]
> do you think it's possible map a single toolbar for multiple tags?

@dmantisk Sounds like you'll have to wait until I add a mapping type specifically for tags. Then you'll be able to list out your tag(s) and pick the toolbar(s) they map to.

Something that might help your use case is the ability to inherit toolbars, which, as I noted in your other request, is on the horizon.

**Comment by chrisgurney** [comment]
Hi @dmantisk! If you're still using Note Toolbar, I'm happy to report the [1.35 beta](https://github.com/chrisgurney/obsidian-note-toolbar/blob/master/src/Help/Releases/en/1.35.md) which includes Rules (beyond just folder mappings) is now [installable via BRAT](https://github.com/chrisgurney/obsidian-note-toolbar#installation).

_If you give it a try, please let me know what you think!_

**Reply by dmantisk** [reply]
Happy to know this feature is now available, but sadly my use case shifted away from this setup so I can't give you proper feedback on it.

Really love your enthusiasm for this project btw.

**Comment by chrsdk** [comment]
> Hi @dmantisk! If you're still using Note Toolbar, I'm happy to report the [1.35 beta](https://github.com/chrisgurney/obsidian-note-toolbar/blob/master/src/Help/Releases/en/1.35.md) which includes Rules (beyond just folder mappings) is now [installable via BRAT](https://github.com/chrisgurney/obsidian-note-toolbar#installation).
> 
> _If you give it a try, please let me know what you think!_

OMG I just tested this and it works! I wanted the exact same thing, a toolbar showing up when I do spaced repetition review, and to my astonishment it is part of the latest release. Thank you so much @chrisgurney for such a wonderful plugin!
